.github/workflows/Build.yml

```diff
@@ -313,7 +324,19 @@ jobs:
       PACKAGE: "formal/symbiyosys"
       OS_NAME: "linux"
       PYTHON_VERSION: "3.8"
-      SKIP: "true"  # See https://github.com/litex-hub/litex-conda-eda/issues/70
+    steps:
+      - uses: actions/checkout@v3
+      - run: apt -qqy update && apt -qqy install ${{ env.APT_REQUIREMENTS }}
+      - uses: ./ci
+
+  symbiyosys-linux-py310:
+    needs: ["yosys-linux-py310"]
+    runs-on: [self-hosted, Linux, X64]
+    container: ubuntu:22.04
+    env:
+      PACKAGE: "formal/symbiyosys"
+      OS_NAME: "linux"
+      PYTHON_VERSION: "3.10"
```

Comment:
I see SymbiYosys still can't be built successfully so I'd say let's keep `SKIP` for jobs with Python 3.7/3.8. Unless building is fixed I'm not sure if adding this job for SymbiYosys+Python 3.10 makes sense. It'd be best to fix it but if not maybe let's add the job with `SKIP: "true"`?

Reply:
Understood, IIRC build failure used to block publication but it doesn't seem to be the case anymore. Maybe we could revisit removing skipped build in a follow up changes (and let them fails until they are fixed)

Reply:
Right, it makes sense to just let them fail. Some of them might have been even fixed upstream already so maybe you could drop all `SKIP`s in a separate commit then?